packages/button/src/Button/Button.tsx

```diff
@@ -0,0 +1,202 @@
+import { Icon as TelegraphIcon } from "@telegraph/icon";
+import { Text as TypographyText } from "@telegraph/typography";
+import clsx from "clsx";
+import React from "react";
+
+import {
+  buttonColorMap,
+  buttonSizeMap,
+  iconColorMap,
+  iconSizeMap,
+  iconVariantMap,
+  textColorMap,
+  textSizeMap,
+} from "./Button.constants";
+
+type RootBaseProps = {
+  variant?: "solid" | "soft" | "outline" | "ghost";
+  size?: "1" | "2" | "3";
+  color?: "accent" | "gray" | "red" | "disabled";
```

Comment:
Yeah disabled is a state but there is a color scheme tied to it (see figma), so it needs to exist alongside the other colors config. You're able to pass the `disabled` prop like normal into the button and it'll work.

Reply:
Accent is the brand color. We're following the same naming convention as Radix Themes. Brand felt too prescriptive of a name where as accent makes more sense. So for us, this would be the knock orange. You can see all of the config for these colors here: https://github.com/knocklabs/telegraph/blob/main/packages/tokens/src/tokens/color.ts